test/neural_net/style_transfer.cxx

```diff
@@ -89,22 +89,138 @@ struct style_transfer_test {
   }
 
   /**
-    Loss
+    Pre Processing
 
-    This test checks whether the Loss out of the total graph defined in TCMPS is
+    This correctness test checks whether the Pre Processing Node present in
+    TCMPS is performing as intended by comparing it to the golden set.
```

Comment:
What will be the general strategy for producing/maintaining the golden set? I'm wary of committing significant binary data to the repo if we can avoid it

Reply:
I'll have a separate files in Python folder that'll generate the golden set.

In the build `YAML` files I'll have a script to get to generate the golden set before the tests are built. That way the golden set doesn't have to directly be committed to the repo. And during the build process it gets generated.